.github/workflows/publish-commit-bottles.yml

```diff
@@ -2,6 +2,10 @@ name: Publish and commit bottles
 
 run-name: "Publish PR #${{ inputs.pull_request }}"
 
+concurrency:
+  group: ${{ github.workflow }}-${{ inputs.pull_request }}
```

Comment:
Looks like `inputs.pull_request` is showing up as blank here, so we're actually running all these jobs serially at the moment.